### PR TITLE
SIMD: SVM fraud proofs

### PR DESCRIPTION
**Pull request by aeyakovenko**
This SIMD proposes a design for adding succinct proofs of invalid computation to Solana's SVM. This would make it much easier for light clients to verify the correctness of transactions, and to detect majority faults.

### CODE REVIEW

File: proposals/0065-svm-fraud-proofs.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+This SIMD proposes a design for adding succinct proofs of invalid
+computation to Solana's SVM. This would make it much easier to
+verify the correctness of transactions, and to detect fraud or
+malicious activity.
+
+
+## Motivation
+
+With fraud proofs, honest participants in the network will be able
+to construct and gossip a fraud proof that can be easily verified
```
Comment (hydrogenbond007) [inline proposals/0065-svm-fraud-proofs.md:24]: Fraud proofs are generally generated when there is an objection on the transaction on L2s which halts that specific block and both the block producer & objector stake some assets then go in proof showdown mode, It makes sense on L2s as their tps is still low and sequencers are the only ones posting proofs. 
--- 
**How would something like this work on Solana and its effects on the wider network?**
Would there be a fork while most of the general ops continue?

**Comment by jacobcreech** [comment]
Hello @aeyakovenko . I'm going to close this SIMD for now as we continue discussions within #proj-spv on the Solana Tech discord. Once there's more to flesh out design-wise, you're welcome to reopen or someone else working in proj-spv can as well with more of the technical documentation on the design.

**Comment by raymond2338** [comment]
Hello, was there any followup work on this? Was looking into this lately and down to contribute

**Review by ZKcash-IrishGov — COMMENTED** [review]
0042 aethens.eth def54ac702f62ad997e931307bc0bc442a2bc16c
0x42d2ae193484470feb0674e6c9f4949b796f5c37
proposals/0042-svm-fraud-proofs.md

**Pull request closed without merging**